dev-libs/zziplib/zziplib-0.13.59.recipe

```diff
@@ -1,31 +1,94 @@
-DESCRIPTION="
-Lightweight library used to easily extract data from files archived in a \
-single zip file.
-"
+SUMMARY="The zziplib extract data from archives in a single zip"
+DESCRIPTION="Lightweight library used to easily extract data from \
+files archived in a single zip file."
 HOMEPAGE="http://zziplib.sourceforge.net/"
-SOURCE_URI="http://sourceforge.net/projects/zziplib/files/zziplib13/0.13.59/zziplib-0.13.59.tar.bz2/download"
+COPYRIGHT="1999-2010 Guido Draheim"
+LICENSE="GNU LGPL v2.1
+	MPL v1.1"
+SOURCE_URI="https://downloads.sf.net/zziplib/zziplib-$portVersion.tar.bz2"
 REVISION="1"
-STATUS_HAIKU="stable"
-DEPEND=""
-CHECKSUM_MD5="14b5a6fc229afe9916d48358479568d3"
-BUILD()
+CHECKSUM_SHA256="cbc90cef60ee498319258e464895f990bb2e6e08f245264d77e44b309c8c218f"
+
+ARCHITECTURES="x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	zziplib$secondaryArchSuffix = $portVersion
+	cmd:zziplib$secondaryArchSuffix = $portVersion
+	cmd:unzip_mem
+	cmd:unzzip
+	cmd:unzzipcat
+	cmd:unzzipcat_mem
+	cmd:unzzipcat_seeko
+	cmd:unzzipdir
+ 	cmd:unzzipdir_mem
+	cmd:zzcat
+	cmd:zzdir
+	cmd:zzxorcat
+	cmd:zzxorcopy
+	cmd:zzxordir
+	lib:libzziplib$secondaryArchSuffix = $portVersion
+	lib:libzzip_0$secondaryArchSuffix  = 13.0.59 compat >=13.0.59
+	lib:libzzip$secondaryArchSuffix = $portVersion
+	lib:libzzipfseeko_0$secondaryArchSuffix = 13.0.59 compat >=13.0.59
+	lib:libzzipfseeko$secondaryArchSuffix = $portVersion
+	lib:libzzipmmapped_0$secondaryArchSuffix = 13.0.59 compat >=13.0.59
+	lib:libzzipmmapped$secondaryArchSuffix = $portVersion
+	lib:libzzipwrap_0$secondaryArchSuffix = 13.0.59 compat >=13.0.59
```

Comment:
If the installation did create this symbolic link:
```
lib/ibzzipwrap-0.so.13 -> ibzzipwrap-0.so.13.0.59
```
then we would have a **`compat >= 13`** for **`lib:libzzipwrap_0$secondaryArchSuffix = 13.0.59`**.
But we don't get any symlink for that lib, as well as for the other libs.
So, if I'm not mistaken, we should drop the **`compat >=13.0.59`** for this lib and the others too.
BTW, while at it, could you also replace the 13.0.59 by $portVersion?